### PR TITLE
Rename some more "features" to "capabilities"

### PR DESCRIPTION
**Pull request by webmaster128**

### CODE REVIEW

File: x/wasm/keeper/addresses_test.go [hunk 1]
```diff
@@ -3,9 +3,9 @@ package keeper
 import (
 	"encoding/json"
 	"fmt"
-	tmbytes "github.com/cometbft/cometbft/libs/bytes"
 	"testing"
 
+	tmbytes "github.com/cometbft/cometbft/libs/bytes"
```
Comment (alpe) [inline x/wasm/keeper/addresses_test.go:8]: This change should not be in here. Please run `make format`

File: x/wasm/keeper/reflect_test.go [hunk 2]
```diff
@@ -24,8 +24,8 @@ import (
 )
 
 const (
-	CyberpunkFeatures = "staking,mask,stargate,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_3,cosmwasm_1_4"
-	ReflectFeatures   = CyberpunkFeatures
+	CyberpunkCapabilities = "staking,mask,stargate,cosmwasm_1_1,cosmwasm_1_2,cosmwasm_1_3,cosmwasm_1_4"
+	ReflectCapabilities   = CyberpunkCapabilities
```
Comment (alpe) [inline x/wasm/keeper/reflect_test.go:28]: 👍

**Review by alpe — APPROVED** [review]
Thanks for the housekeeping. Please run the formatter.

File: x/wasm/keeper/addresses_test.go (hunk 1 above)
Comment (webmaster128) [reply x/wasm/keeper/addresses_test.go:8]: This is the result of `make format`. It's in a separate commit. Feel free to kick it out if needed

**Pull request merged**